src/modules/finance/bitcoin.ts

```diff
@@ -0,0 +1,72 @@
+import type { Casing } from '../string';
+
+/**
+ * The bitcoin address families
```

Comment:
@ST-DDT are our documentation test only verifying method desciption punctuation?

```suggestion
 * The bitcoin address families.
```

Reply:
Currently only modules, their methods, fakers and utilities get checked.
I couldn't find a test that checks for `.` at the end. Maybe that is a lint rule instead?

<details>
<summary>Click for the exact code</summary>

Processed units

https://github.com/faker-js/faker/blob/badaa6d60593fe41a1afa0e9a8bef6ae5bb8a352/scripts/apidocs/generate.ts#L25-L32
https://github.com/faker-js/faker/blob/badaa6d60593fe41a1afa0e9a8bef6ae5bb8a352/scripts/apidocs/processing/class.ts#L68-L72
https://github.com/faker-js/faker/blob/badaa6d60593fe41a1afa0e9a8bef6ae5bb8a352/scripts/apidocs/processing/class.ts#L153-L157
https://github.com/faker-js/faker/blob/badaa6d60593fe41a1afa0e9a8bef6ae5bb8a352/scripts/apidocs/processing/class.ts#L185-L197
https://github.com/faker-js/faker/blob/badaa6d60593fe41a1afa0e9a8bef6ae5bb8a352/scripts/apidocs/processing/class.ts#L97-L107

Description test

https://github.com/faker-js/faker/blob/badaa6d60593fe41a1afa0e9a8bef6ae5bb8a352/test/scripts/apidocs/verify-jsdoc-tags.spec.ts#L60-L74

</details>